packages/hoprd/hopr-admin/pages/index.tsx

```diff
@@ -51,7 +51,7 @@ export default function Home() {
     const api = app.api.apiRef.current
     if (api && app.streamWS.state.status === 'CONNECTED') {
       try {
-        api
+        await api
```

Comment:
```suggestion
        const aliases = await api.getAliases()
        app.updateAliases(() => aliases.json()
```

Reply:
Your approach is mixing async/await and then which seems counter-intuitive. I'd rather stick to one approach.